api/v1alpha4/virtualmachinegroup_types.go

```diff
@@ -81,27 +97,31 @@ type VirtualMachineGroupSpec struct {
 	PowerState VirtualMachinePowerState `json:"powerState,omitempty"`
 
 	// +optional
-	// +kubebuilder:default=TrySoft
+
+	// NextForcePowerStateSyncTime may be used to force sync the power state of
+	// the group to all of its members, by setting the value of this field to
+	// "now" (case-insensitive).
+	//
+	// Please note it is not possible to schedule future syncs using this field.
+	// The only value that users may set is the string "now" (case-insensitive).
+	NextForcePowerStateSyncTime string `json:"nextForcePowerStateSyncTime,omitempty"`
```

Comment:
Please add the comment from the VM `NextRestartTime` about the mutating webhook, ex.

```go
	// A mutating webhook changes this value to the current time (UTC), which
	// the group controller uses to...
```